### DISCUSSION TITLE
小爱音箱 Pro 学习的红外遥控能同步到 HA 吗？

### DISCUSSION THREAD

**Discussion by Bigmonstercai** [XiaoMi/ha_xiaomi_home#250] [Q&A] [answered]
我直接同步好像没有，需要怎么配置吗？

**Comment by SusanPhevos** [comment, marked as answer]
xiaomi_home 基于设备的[MIoT-Spec-V2](https://home.miot-spec.com/spec/xiaomi.wifispeaker.lx06)生成实体，厂商未定义的内容无法生成实体。

**Reply by juzi214032** [reply]
米家是如何调用音箱的红外遥控的呢，通过米家的服务器，然后直接调用音箱的api没有走miot spec协议吗？

xiaomi_home作为官方插件有考虑支持这种非miot协议的功能吗🧐

**Comment by milumet** [comment]
能 我的 他自己就同步进去了

**Reply by juzi214032** [reply]
请问你是什么设备？小爱音箱pro吗

**Reply by NOisi-x** [reply]
小爱play增强版，实测可以，需要在米家app生成虚拟的遥控器

**Reply by juzi214032** [reply]
我是空调伴侣pro，这个好像不行

**Reply by NOisi-x** [reply]
空调伴侣pro可以在米家生成虚拟的设备吗？小爱音箱的红外对完码之后可以直接在米家生成一个虚拟的电视，点开则为红外遥控器的UI，这种情况下Miot auto和米家集成都可以正常识别所有按键功能

**Reply by juzi214032** [reply]
空调伴侣试会在米家生成虚拟设备，但无法同步过去。试了下小爱音箱pro可以同步过去， @topsworld 这两个硬件有什么差别吗？我看miot spec上好像都没有定义红外相关的功能

**Reply by juzi214032** [reply]
试了一下是配对的红外遥控可以同步，通过其他遥控器复制的红外遥控无法同步 @topsworld
